README.md

```diff
@@ -46,243 +14,84 @@ After you're finished please remove all the comments and instructions!
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+# 📖 [Hello_Microverse] <a name="about-project"></a>
 
-**[your_project__name]** is a...
+**[Hello_Microverse]** is my first project in Microverse, done in Block 1 of Module 1.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
```

Comment:
- [ ] Its a good thing you selected the predefined words in the `>` section, but you're meant t write your custom work. kindly indicate the tech stack you used for the project.
## View:
![image](https://user-images.githubusercontent.com/50990925/211818110-ac9e4948-b4b4-4cc5-b176-3005f34e09cc.png)